data_processing/pathology/cli/examples/__init__.py

```diff
@@ -0,0 +1,5 @@
+'''
+Created on April 27, 2021
+
+@author: pashaa@mskcc.org
+'''
```

Comment:
these examples folders are not necessarily packages, but are there to separate the json examples from the python code.